docs/website/docs/dlt-ecosystem/deployments/github-actions.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Github actions
+description: How to run dlt in github actions
+keywords: [dlt, webhook, serverless]
+---
+
+# Native deployment to github actions
```

Comment:
why are we repeating information from the walkthrough instead of improving it?

Reply:
well it is not the same.

Reply:
we need the link to the walkthrough github actions page here

Reply:
we don't have a github actions walkthru yet, but a mention how to do the command and follow instructions